src/pages/en/guides/server-side-rendering.md

```diff
@@ -24,11 +24,19 @@ The following adapters are available today with more to come in the future:
 - [Node.js](/en/guides/integrations-guide/node/)
 - [Vercel](/en/guides/integrations-guide/vercel/)
 
-You can find instructions at the individual adapter links above to complete the following two steps (using `my-adapter` as an example placeholder) to enable SSR.
-1. Install the adapter to your project dependencies via npm or your package manager of choice
+
+You can add any of the official adapters with the following `astro add` command. This will install the adapter and make the appropriate changes to your `astro.config.mjs` file in one step. For example, to install the Netlify adapter, run:
+
+```bash
+npx astro add netlify
+```
+
+You can also add an adapter manually by installing the package and updating `astro.config.mjs` yourself. (See the links above for adapter-specific instructions to complete the following two steps to enable SSR.) Using `my-adapter` as an example placeholder, the instructions will look something like:
+
+1. Install the adapter to your project dependencies  using your preferred package manager. If you’re using npm or aren’t sure, run this in the terminal:
```

Comment:
Extra whitespace
```suggestion
1. Install the adapter to your project dependencies using your preferred package manager. If you’re using npm or aren’t sure, run this in the terminal:
```

Reply:
Thanks, for the awesome catch, Yan! That is exactly where I was pasting the entire 2nd part of that phrase. You caught me! I *thought* I got a single space every time...